app/src/main/res/layout/show_raw_metadata_dialog.xml

```diff
@@ -0,0 +1,227 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
anything in the layout folder will be a view so calling it `layout(View).view_...` is kind of pointless and it's not a fragment so the prefix fragment etc doesn't really apply any helpful information here. the Dialog at the end should be enough to at first glance know that its a view used in other places as a Dialog

Reply:
Looking at the naming otherwise done in the folder it seems however there are some kind of pattern.

You are better than me defining what type of view it is, but to keep consistency I think we should somehow align with the naming already there.

Reply:
chnaged to `view_show_raw_metadata_dialog`